### ISSUE TITLE
About the definition of "service"

### ISSUE THREAD

**Issue by Deguello1** [boucadair/can-framework#14]
> A monolithic function that is provided by a network operator according to a specification. A composite service can be built by orchestrating multiple monolithic services.
This reads strange as the woring mixes function with service. I would encourage some consisitency, e.g., based upon the terminology used by RFC7665 (Service Function). Also, since earlier text mentions service provider, I would suggest s/network operator/service provider.

**Comment by boucadair** [comment]
Fair point. A proposal can be seen at https://github.com/boucadair/can-framework/pull/16/commits/f669192f74c07bc4a3dd40fb95032cdcb2fde2af
